graspy/simulations/sims_sample_corr.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from graspy.simulations import sample_edges
```

Comment:
have you tried doing this import as `from .simulations import sample_edges`?

Reply:
Yeah, I can change it back.